src/doc/rustc/src/platform-support.md

```diff
@@ -260,6 +260,7 @@ target | std | host | notes
 `mips-unknown-linux-uclibc` | ✓ |  | MIPS Linux with uClibc
 [`mips64-openwrt-linux-musl`](platform-support/mips64-openwrt-linux-musl.md) | ? |  | MIPS64 for OpenWrt Linux MUSL
 `mipsel-sony-psp` | * |  | MIPS (LE) Sony PlayStation Portable (PSP)
+`mipsel-sony-psx` | * |  | MIPS (LE) Sony PlayStation 1 (PSX)
```

Comment:
You also need to link the page to it's relevant entry. Like this:
```suggestion
[`mipsel-sony-psx`](platform-support/mipsel-sony-psx.md) | * |  | MIPS (LE) Sony PlayStation 1 (PSX)
```

Reply:
Fixed. Thanks for catching this.

Reply:
huh, looks like the link broke one of the CI checks since the page it links to is added in this PR. It's formatted the same way as the other targets so I'm not sure how to handle this.

Reply:
No idea, sorry. Did you tried running mdbook manually to fix to find the problem ? You could also check if the page is even generated.

Reply:
Turns out I just missed a link in `SUMMARY.md` but it's fixed now.